docs/docs/how-to/setup-intant-on-windows-wsl.mdx

```diff
@@ -0,0 +1,89 @@
+---
+id: setup-instant-on-windows-wsl
+title: How to setup Instant OpenHIE on Windows with WSL
+sidebar_label: Setup instant on Windows
+keywords:
+  - Instant OpenHIE
+  - How to
+  - Windows
+  - WSL
+description: How to setup Instant OpenHIE for development on Windows with WSL
+---
+
+:::info
+The Instant OpenHIE architecture, codebase, and documentation are under active development and are subject to change. While we encourage adoption and extension of the Instant OpenHIE framework, we do not consider this ready for production use at this stage.
+:::
+
+## 1. Install WSL
+Follow the steps on the Microsoft website to install the WSL feature for Windows:
+- [Windows Subsystem for Linux Installation Guide for Windows 10](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+
+## 2. Linux Distribution Setup
+
+If you haven't already done this as part of the step 1 above, you will need to create a user account and password for the Linux Distribution, the following link can assist with that:
+
+- [Create a user account and password for your new Linux distribution](https://docs.microsoft.com/en-us/windows/wsl/user-support)
+
+When using WSL 2 it is better to store files in the Linux subsystem file system rather than in the Windows file system for performance reasons. The easiest way to do this is to simply clone repositories and other files to the $HOME (~) directory on the Linux subsystem.
+
+## 3. Git Setup
+
+Many Linux distributions come with git out of the box, but if required, the following commands will install git:
+
+```sh
+sudo apt-get update
+```
+```sh
+sudo apt-get install git
+```
+
+## 4. Docker Desktop Setup
+
+Download and install Docker Desktop from:
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
```

Comment:
Help out Linux users.
I tried user the terminal cli - it's a schlep so even though I don't use the gui the setup is great

```suggestion
> Docker Desktop install `docker-compose` by default and sets up docker without sudo in the Linux terminal

```